api/v1alpha4/types.go

```diff
@@ -117,7 +117,7 @@ type DOLoadBalancer struct {
 	// +optional
 	// +kubebuilder:validation:Minimum=1
 	// +kubebuilder:validation:Maximum=65535
-	Port int `json:"port,omitempty"`
+	Port int32 `json:"port,omitempty"`
```

Comment:
Linter was complaining about `int` -> `int32` conversion from capdo port to capi port, as an "integer overflow" issue, which of course, wouldn't happen realistically with ports, but seemed not a bad idea to change this port to int32 to match upstream.